viite-UI/src/view/LinkPropertyLayer.js

```diff
@@ -761,6 +763,10 @@
           feature.roadLinkData = road;
           simulatedOL3Features.push(feature);
           afterTransferLinks.push(road);
+          var marker = cachedMarker.createMarker(feature.roadLinkData);
+          if(map.getView().getZoom() > zoomlevels.minZoomForDirectionalMarkers) {
+            simulatedRoadsLayer.getSource().addFeature(marker);
```

Comment:
Edit: Reviewed with with dev, and should be simulatedRoadsLayer like it is now since the Siirra only simulates the after transfer. (Y)